types/dropzone/index.d.ts

```diff
@@ -26,13 +27,22 @@ declare namespace Dropzone {
         trgHeight?: number;
     }
 
+    export interface DropzoneFileUpload {
+        progress: number;
+        total: number;
+        bytesSent: number;
+        uuid: string;
+        totalChunkCount?: number;
```

Comment:
I don't see `totalChunkCount` in doc
Can you explain more?

Reply:
The docs are not very complete; I don't think they were fully updated after the chunked uploads feature was added.  There are actually a few more fields in there.  The contents of `progress` doesn't seem to make sense, and `chunked: boolean` only appears when running in chunked mode.

![Screen Shot 2020-05-20 at 9 19 31 AM](https://user-images.githubusercontent.com/53452532/82471142-26c01980-9a7b-11ea-99c3-b59d12fc1c3c.png)

For a non-chunked upload, the fields are `bytesSent`, `filename`, `progress`, `total`, and `uuid`:

![Screen Shot 2020-05-20 at 9 24 17 AM](https://user-images.githubusercontent.com/53452532/82471564-b2d24100-9a7b-11ea-9a8d-86e364f0fdff.png)